### ISSUE TITLE
Dynamic input cant disable create buttom

### ISSUE THREAD

**Issue by funkindy** [tusen-ai/naive-ui#4400]
### TuSimple/naive-ui version (版本)
2.34.3

### Vue version (Vue 版本)
3.2.45

### Browser and its version (浏览器及其版本)
109.0.5414.75

### System and its version (系统及其版本)
Win 10

### Reappearance link (重现链接)
https://codesandbox.io/s/unruffled-lena-4z7gu0?file=/src/App.vue

### Reappearance steps (重现步骤)
Pass `{disabled: true}` to dynamic input's `create-button-props`

### Expected results (期望的结果)
Create button is disabled and cant be clicked

### Actual results (实际的结果)
Create button is active

### Remarks (补充说明)
Passing other props, e.g. {type: "info"} works as expected and are being passed to the button correctly

**Comment by OrbisK** [comment]
![image](https://user-images.githubusercontent.com/37191683/214596613-48c0a4ab-1103-47b0-bd93-a755e60b2da2.png)

That's the reason. You can't override `theme`, `themeOverrides`, `onClick` and `disabled`.

Not sure why disabled is in that list. Maybe some side effects,

**Comment by funkindy** [comment]
Yes, its a little bit weird to have an active button on a huge disabled form.

**Comment by jahnli** [comment]
This issue does not have any recent activity. If you are still experiencing similar problems, open a new error, including a minimal copy of the problem
